### PR TITLE
Screen/result

### PR DESCRIPTION
**Pull request by sanjitsarkar**
Added Result Screen

### CODE REVIEW

File: category.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        <a href="index.html" class="text-xl">ZenTrivia</a>
+    </header>
+    <main class="container h-full w-full col items-center">
+        <section class="category-section w-full text-light">
+            <h1 class="text-3xl text-bold mb-4 text-center">Choose <span class="text-secondary">Trivia</span> Category
+            </h1>
+            <div class="row flex-gap-1 justify-center">
+                <div class="card w-60 bg-primary">
+                    <div class="card-header">
+                        <img src="images/classical.png" alt="" class="img h-40 w-full cover">
```
Comment (DivyanshSareen) [inline category.html:25]: empty alt

**Review by DivyanshSareen — APPROVED** [review]
Looks good to me

**Comment by manojncsoni007** [comment]
looks good

**Review by manojncsoni007 — COMMENTED** [review]
looks good to me

**Pull request merged**